packages/mdc-tab-scroller/_mixins.scss

```diff
@@ -26,10 +26,28 @@
 @import "@material/feature-targeting/mixins";
 @import "./variables";
 
+@mixin mdc-tab-scroller-transition(
```

Comment:
Can you merge this to `develop` branch. This is the branch which gets synced to google3/.

Reply:
It looks like develop is currently behind? It's missing my previous PR #5142 which is causing merge conflicts.

Reply:
Just synced your changes to develop.

`develop` is always ahead of `master`. Can you please rebase your branch with origin/develop? Thanks!

Reply:
Done!